Common/src/linear_algebra/CSysSolve.cpp

```diff
@@ -192,7 +192,7 @@ unsigned long CSysSolve<ScalarType>::CG_LinSolver(const CSysVector<ScalarType>&
                                                   const CPreconditioner<ScalarType>& precond, ScalarType tol,
                                                   unsigned long m, ScalarType& residual, bool monitoring,
                                                   const CConfig* config) const {
-  const bool master = (SU2_MPI::GetRank() == MASTER_NODE) && (omp_get_thread_num() == 0);
+  const bool masterRank = (SU2_MPI::GetRank() == MASTER_NODE);
```

Comment:
Using master constructs instead of tests against the thread number makes it easier to track serial parts of the code in performance tests.